systers_portal/common/tests/selenium/test_can_goto_common_pages.py

```diff
@@ -0,0 +1,27 @@
+from common.tests.selenium.base import SeleniumTestCase
+
+
+class TestCanGoToCommonPages(SeleniumTestCase):
```

Comment:
Just a suggestion, to maintain consistency, you should a class level docstring here as well :)

Reply:
Yeah noted... Very important!